data/json/items/tool/workshop.json

```diff
@@ -965,8 +965,8 @@
   {
     "id": "robofac_gun_kit",
     "type": "TOOL",
-    "name": { "str": "Hub 01 Hybrid maintenance kit" },
-    "description": "This is a heavy-duty bag marked with the Hub 01 logo.  It contains space for barrels, adaptors and proprietary tools.  Embossed on the inside is a sketch of the Hybrid Weapons Platform.",
+    "name": { "str": "Hub 01 hybrid maintenance kit" },
```

Comment:
I had also noticed that issue when translating, I initially thought about making it lowercase (which I noted on Transifex), but then I figured that maybe having the whole _Hybrid Weapons Platform_ or shorthand _HWP_ in the name instead of it would be better idea?

e.g. Hybrid Weapons Platform maintenance kit?

What do you think? cc @bombasticSlacks

Reply:
it's getting a little long with the full "Hybrid Weapons Platform". Though that seems to be how all of these verbose Hub acronym items are. I would lean towards HWP over my pointlessly capitalized "Hybrid"